assembl/static2/js/app/components/harvesting/harvestingBox.jsx

```diff
@@ -82,10 +83,13 @@ class DumbHarvestingBox extends React.Component<Props, State> {
     super(props);
     const { extracts, cancelHarvesting } = this.props;
     const hasExtract = extracts ? extracts.length > 0 : false;
-    const extract = this.getCurrentExtract(0);
+    const { hash } = window.location;
+    const hashExtractId = hash !== '' ? hash.split('#')[2] : null;
+    const extractIndex = hashExtractId && extracts ? extracts.indexOf(this.getCurrentExtractById(hashExtractId)) : 0;
```

Comment:
`this.getCurrentExtractById(hashExtractId)` may return null if hashExtractId is not found.

Reply:
[].indexOf(null) returns -1
you may set extractIndex to 0 if you get a -1

Reply:
done